include/SFML/System/Vector2.hpp

```diff
@@ -73,6 +73,14 @@ class Vector2
     ////////////////////////////////////////////////////////////
     template <typename U>
     constexpr explicit Vector2(const Vector2<U>& vector);
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Construct a unit vector from an angle
+    ///
+    /// \param angle Angle
+    ///
+    ////////////////////////////////////////////////////////////
+    explicit Vector2(const Angle& angle);
```

Comment:
I'd probably pass `Angle` by value and name it `phi` like here:
https://github.com/SFML/SFML/blob/c7327353b3eaa1ac62c13195a1424033121fb5de/include/SFML/System/Vector2.hpp#L140

(also applies once the constructor signature changes)

Reply:
Weird, the implementation uses the name `angle`. I'll use `phi` if you want though.

https://github.com/SFML/SFML/blob/3ae85854e4674debe3b30ae2f6cf9d9926738789/include/SFML/System/Vector2.inl#L110

Reply:
Oh, that's an oversight, feel free to fix it. Ultimately it's not _that_ important 🙂

Reply:
Might be worth some library-wide rename alongside changing many `const Angle&` params to `Angle`. For a later PR but I'll make a note to check for consistent `sf::Angle` use.

Reply:
I was mostly thinking about calling an angle `phi` in the context of polar coordinates (i.e. a vector) -- i.e. in cases where there is another coordinate `r` (radius). Not sure if that necessarily applies to the rest of the code base; so `angle` might still be fine elsewhere.